resource_booking/models/resource_booking.py

```diff
@@ -100,14 +109,13 @@ class ResourceBooking(models.Model):
         compute="_compute_start",
         copy=False,
         index=True,
-        inverse="_inverse_dates",
+        readonly=False,
```

Comment:
Haven't you mixed these changes in an unrelated commit?

Reply:
AFAICS it's OK. I'm removing the inverse, but still allowing writes with readonly=False.

Reply:
I'm saying that the changes are on the second commit, while the topic belongs to the first commit.

Reply:
I double-checked it and it's OK. The need to change from `_inverse_dates` to `_sync_dates` happened in 1cd849d8c40f644aefb62391ce0c3dfe6dd78012.